feature-group-definitions/web-midi.yml

```diff
@@ -2,3 +2,53 @@ name: Web MIDI
 description: The Web MIDI API enables selecting MIDI input and output devices and sending and receiving MIDI messages.
 spec: https://webaudio.github.io/web-midi-api/
 caniuse: midi
+compat_features:
+  - api.MIDIAccess
+  - api.MIDIAccess.inputs
+  - api.MIDIAccess.outputs
+  - api.MIDIAccess.statechange_event
+  - api.MIDIAccess.sysexEnabled
+  - api.MIDIConnectionEvent
+  - api.MIDIConnectionEvent.MIDIConnectionEvent
+  - api.MIDIConnectionEvent.port
+  - api.MIDIInput
+  - api.MIDIInput.midimessage_event
+  - api.MIDIInputMap
+  - api.MIDIInputMap.entries
+  - api.MIDIInputMap.forEach
+  - api.MIDIInputMap.get
+  - api.MIDIInputMap.has
+  - api.MIDIInputMap.keys
+  - api.MIDIInputMap.size
+  - api.MIDIInputMap.values
+  - api.MIDIInputMap.@@iterator
+  - api.MIDIMessageEvent
+  - api.MIDIMessageEvent.MIDIMessageEvent
+  - api.MIDIMessageEvent.data
+  - api.MIDIOutput
+  - api.MIDIOutput.send
+  - api.MIDIOutputMap
+  - api.MIDIOutputMap.entries
+  - api.MIDIOutputMap.forEach
+  - api.MIDIOutputMap.get
+  - api.MIDIOutputMap.has
+  - api.MIDIOutputMap.keys
+  - api.MIDIOutputMap.size
+  - api.MIDIOutputMap.values
+  - api.MIDIOutputMap.@@iterator
+  - api.MIDIPort
+  - api.MIDIPort.close
+  - api.MIDIPort.connection
+  - api.MIDIPort.id
+  - api.MIDIPort.manufacturer
+  - api.MIDIPort.name
+  - api.MIDIPort.open
+  - api.MIDIPort.state
+  - api.MIDIPort.statechange_event
+  - api.MIDIPort.type
+  - api.MIDIPort.version
+  - api.Navigator.requestMIDIAccess
+  # Later additions:
```

Comment:
I suppose we had to bring all of the BCD keys in this file and comment out a few because we don't have #841 yet. Is that it?

Reply:
Kind of. If we tagged all of Web MIDI in BCD with "web-features:web-midi" then we would need a mechanism like https://github.com/web-platform-dx/web-features/issues/841.

But after all of the recent work my idea of the ideal design has shifted somewhat, and https://github.com/web-platform-dx/web-features/issues/979 is a path to giving individual BCD keys a Baseline status separate from the overall feature's status. https://github.com/web-platform-dx/web-features/issues/866#issuecomment-2078907927 is a mockup of what I'd like to see when reviewing a feature.

So I think that the way to resolve these "later additions" comments is likely to tag everything in BCD, review the groups of statuses that result, and then designate one of those groups (usually the earliest) as the status for the overall feature.